tracepointdebug/probe/trace_point_manager.py

```diff
@@ -90,6 +93,21 @@ def expire_trace_point(self, trace_point):
                 self._trace_points.pop(trace_point_id).remove_trace_point()
 
     def publish_event(self, event):
+        try:
+            if self.broker_manager._tracepoint_data_redaction_callback and isinstance(event, TracePointSnapshotEvent):
+                trace_redaction = {
+                    "file_name": event.file,
+                    "line_no": event.line_no,
+                    "method_name": event.method_name,
+                    "frames": event.frames
+                }
+                self.broker_manager._tracepoint_data_redaction_callback(trace_redaction)
+                event.file = trace_redaction["file_name"]
+                event.line_no = trace_redaction["line_no"]
+                event.method_name = trace_redaction["method_name"]
```

Comment:
Do we expect masking method name so we are getting it again from redacted trace?